<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>791</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Get</string>
     </attribute>
     <widget class="QTextBrowser" name="textGet">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>721</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>261</width>
        <height>29</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QSpinBox" name="spnIdGet">
       <property name="maximum">
        <number>100000</number>
       </property>
      </widget>
      <widget class="QPushButton" name="btnGet">
       <property name="text">
        <string>get</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>List</string>
     </attribute>
     <widget class="QPushButton" name="btnList">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>list</string>
      </property>
     </widget>
     <widget class="QListWidget" name="textList">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>721</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Create</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>731</width>
        <height>431</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtNameCreate"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Author:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtAuthorCreate"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Genre:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtGenreCreate"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Release Date:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateAlbumCreate"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Songs quantity: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spnSongsCreate">
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QTextBrowser" name="textCreate"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btnCreate">
         <property name="text">
          <string>create</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Update</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>731</width>
        <height>431</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QTextBrowser" name="textUpdate"/>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Album ID:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spnIdUpdate">
             <property name="maximum">
              <number>100000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtNameUpdate"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Author:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtAuthorUpdate"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Genre:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtGenreUpdate"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Release Date:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateAlbumUpdate"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Songs quantity: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spnSongsUpdate">
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btnUpdate">
         <property name="text">
          <string>update</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Delete</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>303</x>
        <y>140</y>
        <width>311</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Album ID:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spnIdDelete">
             <property name="maximum">
              <number>100000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btnDelete">
           <property name="text">
            <string>delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextBrowser" name="textDelete"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>17</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
